scormxblock/scormxblock.py

```diff
@@ -36,13 +36,13 @@ def s3_upload(all_content, temp_directory, dest_dir):
     """
     Actual handling of the s3 uploads.
     """
-    session = boto3.Session(
-        aws_access_key_id=settings.DJFS.get('aws_access_key_id'),
-        aws_secret_access_key=settings.DJFS.get('aws_secret_access_key'),
-        region_name=settings.DJFS.get('region_name'),
```

Comment:
Why are we removing this variable? this would be breaking the functionality for non CEPH installations, right? Out changes should not affect our installations that are using S3